website_form_recaptcha/models/website_config_settings.py

```diff
@@ -1,15 +1,34 @@
 # Copyright 2019 Simone Orsi - Camptocamp SA
 # License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
 
-from odoo import fields, models
+from odoo import api, fields, models
 
 
 class ResConfigSettings(models.TransientModel):
     _inherit = 'res.config.settings'
 
     recaptcha_key_site = fields.Char(
         related='website_id.recaptcha_key_site',
+        readonly=False,
```

Comment:
Related fields not readonly are a good idea? Or transient models are the exception? cc @pedrobaeza

Reply:
This is on purpose, as this is for the settings part, where the change is "transferred" to the corresponding model, so this is totally correct.